### DISCUSSION TITLE
Getting AppInsight Error when opening latest version of Fetch Xml 1.2024.4.1

### DISCUSSION THREAD

**Discussion by TristonArisawa** [rappen/FetchXMLBuilder#1056] [Q&A]
Getting AppInsight Error when opening latest version of Fetch Xml 1.2024.4.1
Incidentally started after the CrowdStrike issue 7/19/2024

**Comment by rappen** [comment]
Interesting!
Can you find some more details?
I can't find any errors... :-/

**Comment by TristonArisawa** [comment]
Here is a screenshot:
***@***.***
So this happened recently after some security update, but this is only one of a couple of toolbox plugins that don’t work anymore. I do know that github I blocked on this server.

**Reply by rappen** [reply]
Thanks @TristonArisawa 
Unfortunately, I can't get the screenshot here on GitHub, and it's not even in the notification email.

Is it still a problem for you? Is FetchXML Builder not possible to start? Can you send the names of any other tools with the same issues?
